js/app1.js

```diff
@@ -1,121 +1,225 @@
+  }
 }
 
-let askCountries = prompt('Can you guess how many countries I have lived in? 1-5.');
-if (askCountries <= '3') {
-  alert('You are incorrect! The answer is 4.');
-  count = count - 1;
-} else if (askCountries === '4') {
-  alert('You are correct!');
-  count = count + 1;
-} else if (askCountries === '5') {
-  alert('You went too far ' + quest1 + '! The asnwer is 4.');
-  count = count - 1;
+function askDestinations(counter){
+  let askCountries = prompt('Can you guess how many countries I have lived in? 1-5.');
+  if (askCountries <= '3') {
+    alert('You are incorrect! The answer is 4.');
+  } else if (askCountries === '4') {
+    alert('You are correct!');
+    counter = counter + 1;
+  } else if (askCountries >= '5') {
+    alert('You went too far ' + quest1 + '! The answer is 4.');
+  }
+  return counter;
 }
 
-let askCayman = prompt('Is the following statement true? I live in the Cayman Islands but I do not like the beach.');
-askCayman = askCayman.toLowerCase();
-
-if (askCayman === 'yes' || askCayman === 'y') {
-  //console.log(askCayman);
-  alert('Crazy, but you are right ' + quest1 + '! I live in an island but I do not like the beach!');
-  count = count + 1;
-} else if (askCayman === 'no' || askCayman === 'n') {
-  //console.log(askCayman);
-  alert('It is actually true ' + quest1 + '! Does not make any sense, does it?');
-  count = count - 1;
-} 
-
-let askStingray = prompt('Is the following statement true? I have a pet stingray.');
-askStingray = askStingray.toLowerCase();
-
-if (askStingray === 'yes' || askStingray === 'y') {
-  //console.log(askStingray);
-  alert('That does not make any sense ' + quest1 + '! But I have petted and kissed many!');
-  count = count - 1;
-} else if (askStingray === 'no' || askStingray === 'n') {
-  //console.log(askStingray);
-  alert('Correct, ' + quest1 + '! But I have petted and kissed many!');
-  count = count + 1;
-} 
-
-let askBrain = prompt('Is the following statement true? My brain physically hurts when learning JavaScript.');
-askBrain = askBrain.toLowerCase();
-
-if (askBrain === 'yes' || askBrain === 'y') {
-  //console.log(askBrain);
-  alert('You are unfortunately right ' + quest1 + '! JS makes parts of my brain that I did not even know I had ache!');
-  count = count + 1;
-} else if (askBrain === 'no' || askBrain === 'n') {
-  //console.log(askBrain);
-  alert('I wish that was true ' + quest1 + '. I wish that was true...');
-  count = count - 1;
+// function askDestinations(){
+//   let askCountries = prompt('Can you guess how many countries I have lived in? 1-5.');
+//   if (askCountries <= '3') {
+//     alert('You are incorrect! The answer is 4.');
+//   } else if (askCountries === '4') {
+//     alert('You are correct!');
+//     return 1;
+//   } else if (askCountries >= '5') {
+//     alert('You went too far ' + quest1 + '! The answer is 4.');
+//   }
+//   return 0;
+// }
+
+// count += askDestinations();
```

Comment:
Since this is almost exactly the same as your other askDestinations function, it can be safely removed at this point.